### DISCUSSION TITLE
Can seq use connections strings to auth with Azure eventhub?

### DISCUSSION THREAD

**Discussion by mpp-olano** [datalust/seq-tickets#1749] [Help]
Can seq use connections strings to auth with Azure eventhub?

**Comment by nblumhardt** [comment]
Hi! The _Seq.Input.AzureEventHub_ app appears to be connection-string based:

https://github.com/Hinni/Seq.Input.AzureEventHub/blob/master/src/Seq.Input.AzureEventHub/AzureEventHubInput.cs#L20

Let me know if this helps,
Nick

**Comment by mpp-olano** [comment]
Hi Nicholas,

Yes, it helpful.

Thanks,



[Icon  Description automatically generated]
Olan Odunlami
DevOps Engineer
Email: ***@***.***
www.mypinpad.com<http://www.mypinpad.com/> | LinkedIn<https://www.linkedin.com/company/mypinpad-ltd/> | Twitter<https://twitter.com/MYPINPAD>

MYPINPAD Ltd is a limited company registered in England and Wales, company number 08100986. Registered office: 3 Assembly Square, Britannia Quay, Cardiff Bay, Cardiff, CF10 4PL. For information about MYPINPAD as a company and how we process data and monitor communications please see our website<https://mypinpad.com/about/> and our Privacy Policy<https://mypinpad.com/privacy-policy/>.

From: Nicholas Blumhardt ***@***.***>
Sent: 26 January 2023 22:16
To: datalust/seq-tickets ***@***.***>
Cc: Olan Odunlami ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [datalust/seq-tickets] Can seq use connections strings to auth with Azure eventhub? (Discussion #1749)

Caution: This email originated from outside of the organisation. Please take care when clicking links or open attachments unless you recognise the sender and know the content is safe. If in doubt contact IT Support.


Hi! The Seq.Input.AzureEventHub app appears to be connection-string based:

https://github.com/Hinni/Seq.Input.AzureEventHub/blob/master/src/Seq.Input.AzureEventHub/AzureEventHubInput.cs#L20

Let me know if this helps,
Nick
